src/constructor.io/models/Quizzes/QuizRequest.cs

```diff
@@ -0,0 +1,135 @@
+﻿using System;
+using System.Collections;
+using System.Collections.Generic;
+using Constructorio_NET.Utils;
+using Newtonsoft.Json;
+
+namespace Constructorio_NET.Models
+{
+    /// <summary>
+    /// Constructor.io QuizRequest Request Class.
+    /// </summary>
+    public class QuizRequest
+    {
+        /// <summary>
+        /// Gets or sets the quiz id used to get quiz result.
+        /// </summary>
+        public string Id { get; set; }
+
+        /// <summary>
+        /// Gets or sets the name of the section.
+        /// </summary>
+        public string Section { get; set; }
+
+        /// <summary>
+        /// Gets or sets An array of answers in the format [[1,2],[1]].
+        /// </summary>
+        public List<List<string>> Answers { get; set; }
+
+        /// <summary>
+        /// Gets or sets version identifier for the quiz.
+        /// </summary>
+        public string VersionId { get; set; }
+
+        /// <summary>
+        /// Gets or sets user test cells.
+        /// </summary>
+        public Dictionary<string, string> TestCells { get; set; }
```

Comment:
Not sure if we should include section/test cells here.

For section, I don't think there's currently a way to create quizzes for other sections.

For test cells, I don't think the api actually does anything with it.